packages/testing/src/common.cairo

```diff
@@ -71,3 +71,13 @@ pub fn assert_entrypoint_not_found_error<T, +Drop<T>>(
         panic!("{selector} call was expected to fail, but succeeded");
     }
 }
+
+/// Creates and returns a new array consisting of the given `element` repeated `n_times` times.
+pub fn repeat<T, N, +Copy<T>, +Drop<T>, +Into<N, u256>>(element: T, n_times: N) -> Array<T> {
```

Comment:
I don't think we should export this as part of our external API from testing, it should be an internal test_common item instead. We don't want the testing API to grow to big, so we should include utilities that will be potentially often used, and I think this is not the case.